app/Models/ModelHasLabel.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
This file should be a trait, something like `HasLabels` so that we can add it to models that need labels.
Should work similar to https://github.com/spatie/laravel-permission/blob/main/src/Traits/HasRoles.php

Reply:
Should be covered by latest commits